src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-wizard/rgw-multisite-wizard.component.html

```diff
@@ -251,3 +208,124 @@
     </ng-container>
   </cd-modal>
 </div>
+
+<ng-template #nonMultiClusterTemplate>
+  <ng-container *ngIf="!loading; else loadingTemplate">
+    <ng-container *ngIf="!setupCompleted else exportTokenTemplate">
+      <ng-container *ngTemplateOutlet="reviewTemplate"></ng-container>
+    </ng-container>
+  </ng-container>
+</ng-template>
+
+<ng-template #loadingTemplate>
+  <ng-container *ngTemplateOutlet="progressTemplate"></ng-container>
+</ng-template>
+
+<ng-template #progressCompleteTemplate>
+  <div *ngIf="isMultiClusterConfigured; else exportTokenTemplate">
+    <div class="text-center text-success"
+         i18n>
+      Multi-site replication setup is complete.
+    </div>
+  </div>
+</ng-template>
+
+<ng-template #progressTemplate>
+  <cd-progress  [progress]="executingTask?.progress"
+                [executingTaskName]="executingTask?.name?.replace('progress/', '')">
+  </cd-progress>
```

Comment:
We can manage these two values easily in our component and then pass the two to progress.
This will also help us to maintain and debug
```suggestion
  <cd-progress  [state]="state"
                            [label]="currentLabel">
  </cd-progress>
```